gradle/java-module.gradle

```
@@ -523,7 +523,7 @@ checkerFramework {
 	extraJavacArgs = [
 	        '-AsuppressWarnings=initialization',
 			"-Astubs=${project.rootDir}/checkerstubs",
-			'-AonlyDefs=^org\\.hibernate\\.jpamodelgen\\.'
+			'-AonlyDefs=^org\\.hibernate\\.jpamodelgen\\.,^org\\.hibernate\\.spi\\.'
```

AFAICS from the documentation, the whole argument is a single regex, so you'd need this instead:

```suggestion
			'-AonlyDefs=^org\\.hibernate\\.(jpamodelgen|spi)\\.'
```